### DISCUSSION TITLE
Fake GC-01: CACHIP datasheet

### DISCUSSION THREAD

**Discussion by KountKartoffel** [Gissio/radpro#67] [Ideas]
Unfortunately I bought what appears to be a fake GC-01 detector last year and confirmed that it has the CACHIP MCU. In another thread  (https://github.com/Gissio/radpro/issues/48) I read that there was no publicly  available technical documentation, however I found this document on the website https://www.hqonline.com/product-detail/8-bit-mcu-microcontrollers-cachip-ca51f251l2-2500304728.

Most of it appears to be in Chinese however if someone can translate (or machine translate) the document, I would hope it may help to figure out how to make the software compatible with this device.

[CACHIP DataSheet.pdf](https://github.com/Gissio/radpro/files/15057950/CACHIP.DataSheet.pdf)

**Comment by Gissio** [comment]
This datasheet seems to be for a different MCU.

**Reply by kelogs** [reply]
Any luck with it ?
